easybuild/easyconfigs/b/Blitz++/Blitz++-1.0.2-foss-2020a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'Blitz++'
+version = '1.0.2'
+
+homepage = 'https://github.com/blitzpp/blitz'
+
+description = """
+ Blitz++ is a (LGPLv3+) licensed meta-template library for array manipulation
+ in C++ with a speed comparable to Fortran implementations, while preserving an
+ object-oriented interface
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = [('https://github.com/blitzpp/blitz/archive/')]
+sources = ['%(version)s.tar.gz']
+checksums = ['500db9c3b2617e1f03d0e548977aec10d36811ba1c43bb5ef250c0e3853ae1c2']
+
+builddependencies = [('CMake', '3.16.4')]
+
+configopts = '--enable-shared'
```

Comment:
That doesn't look like a CMake compatible flag at all

Reply:
Thanks @Micket, I suspect that was hanging around from before the more to `CMake`.  It's gone now.